src/main/webapp/ts/matches.js

```diff
@@ -146,10 +146,6 @@ function displayNewMatchPopup(matches) {
         var jsonString = JSON.stringify(match);
         var newOption = new Option(matchString, jsonString);
         newMatchContainer.add(newOption, undefined);
-        // const optionElement = document.createElement('option');
```

Comment:
This commented out section will confuse readers who want to modify the code. Is the commented out code needed or can it be removed? 

Please try to do a self code-review to try to catch these types of issues before submitting code for review.

Reply:
I don't know where that came from. The commented out code happened when I ran tsc command. All I changed was two lines in matches.ts

Reply:
The commented out section is being removed in that commit, so it's gone already.
It got removed bc it no longer existed in the matches.ts file. So no removal needed to be done on my end.

Reply:
ah I see - that makes more sense. Sounds good.